src/components/core/core.js

```diff
@@ -63,14 +62,30 @@ export default class Core extends UIObject {
     return this.getPlugin('strings') || { t: (key) => key }
   }
 
+  get mediaControl() {
+    return this.getPlugin('media_control') || this.dummyMediaControl
+  }
+
+  get dummyMediaControl() {
+    return new UICorePlugin(this)
+  }
+
+  get activeContainer() {
```

Comment:
The `activeContainer` is a new interface, create to replace the `getCurrentContainer` function. This one should be documented.